change_of_vars_demo1d/change_of_vars_demo1d.ipynb

```diff
@@ -0,0 +1,274 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #43.      savefig("/content/change_of_vars_demo1d_latexified.pdf") 
</pre><p>Once you setup terminal, fork the main repo and do that all work in that, so you can give actual path need to give to save figure. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/7/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>